tests/pass/stacked-borrows/stack-printing.stdout

```diff
@@ -0,0 +1,5 @@
+0..1: [ SharedReadWrite<TAG> ]
```

Comment:
Might be worth adding a comment that left is the bottom of the stack?

Also please add a test that involves and unknown bottom, to see how that prints.